enterprise/frontend/src/metabase-enterprise/moderation/components/ModerationStatusIcon/ModerationStatusIcon.tsx

```diff
@@ -13,8 +13,25 @@ export const ModerationStatusIcon = ({
   filled = false,
   ...iconProps
 }: ModerationStatusIconProps) => {
-  const { name: iconName, color: iconColor } = getStatusIcon(status, filled);
-  return iconName ? (
-    <Icon name={iconName} color={color(iconColor)} {...iconProps} />
-  ) : null;
+  const { name: iconName, color: iconColor } = getStatusIcon(
+    status ?? null,
+    filled,
+  );
+  if (!iconName) {
+    return null;
+  }
+  const reformattedIconProps = {
+    ...iconProps,
+    size:
+      typeof iconProps.size === "string"
+        ? parseInt(iconProps.size)
+        : iconProps.size,
+  };
+  return (
+    <FixedSizeIcon
+      name={iconName}
+      color={color(iconColor)}
+      {...reformattedIconProps}
+    />
+  );
```

Comment:
I've added `FixedSideIcon` here, in place of `Icon`, so that the icon doesn't shrink when the viewport is narrow